app/routes.py

```diff
@@ -1,24 +1,50 @@
-from app import app
-from app.controllers import HomeController, LoginController, LoginPageController, OMDBSearchController
+from app import app, BUILD_DIR
+from app.controllers import (HomeController, LoginController,
+                             LoginPageController, OMDBSearchController)
+from app.models import User
+from app.session_util import login_required
 
-from flask import request
+from flask import jsonify, send_from_directory
 
-from app.session_util import login_required
 
-@app.route('/')
-@login_required
-def home():
-    return HomeController().handle()
+# For all routes, return index.html (so that React Router can handle routing).
+# Flask matches the most specific routes first, so this will only be called
+# if nothing else matches.
+# TODO: Add all login_required logic back
+@app.route('/', defaults={'path': ''})
+@app.route('/<path:path>')
+def serve(path):
+    return send_from_directory(BUILD_DIR, 'index.html')
+
+
+@app.route('/current-user')
+def get_current_user():
+    user = User.query.first()
```

Comment:
Lol I guess this can get replaced with the login logic once we have that back

Reply:
Yeah this was the original logic for the homepage (I'm  sure it's out of date now)